content/kubernetes/deployment/openshift/openshift-cli.md

```diff
@@ -182,7 +182,8 @@ Each Redis Enterprise cluster requires at least 3 nodes. Single-node RECs are no
 1. Patch the validating webhook with the certificate.
 
     ```sh
-    kubectl patch ValidatingWebhookConfiguration redis-enterprise-admission --patch "$(cat modified-webhook.yaml)"
+    oc patch ValidatingWebhookConfiguration /
+    redis-enterprise-admission --patch "$(cat modified-webhook.yaml)"
```

Comment:
1. Use a backslash (and not a forward slash) to split a command over multiple lines.
2. Multi line commands are typically indented from the 2nd line and on.
3. Repeat over additional occurrences below.
```suggestion
    oc patch ValidatingWebhookConfiguration \
        redis-enterprise-admission --patch "$(cat modified-webhook.yaml)"
```

Reply:
@zcahana if I indent the second line, will it still work to copy and paste into the terminal?

Reply:
Definitely.